README.md

```diff
@@ -26,17 +27,17 @@ A sample template is provided in [`keys.template.json`](./keys.template.json), a
 
 **NOTE**: You should only specify the browser you wish to publish to. If there are any invalid configuration, the action will fail! I.e, no empty key allowed such as `"chrome": {}`.
 
-Copy the content of your `keys.json` into a github secret with the name `BPP_KEYS`. Then, the action can be used as follows:
+Copy the content of your `keys.json` into a github secret with a name of your choosing, in this case we used `BPP_KEYS`. Then, the action can be used as follows:
 
 ```yaml
 steps:
   - name: Browser Plugin Publish
-    uses: plasmo-corp/bpp@v1
+    uses: plasmo-corp/bpp@v2
     with:
       keys: ${{ secrets.BPP_KEYS }}
 ```
 
-**NOTE**: If you're publishing to Opera or Edge, you will need to setup chromium for puppeteer before running `bpp`. In v2, we will either deprecate puppeteer approach altogether (since it's quite a leaky abstraction), or have an option for bpp to download and setup the browser for you.
+**NOTE**: If you're publishing to the Opera store, you will need to setup chromium for puppeteer before running `bpp`. In v3, we will hopefully deprecate puppeteer approach altogether (since it's quite a leaky abstraction), or have an option for bpp to download and setup the browser for you.
```

Comment:
```suggestion
**NOTE**: If you're publishing to the Opera store, you will need to set up Chromium for puppeteer before running `bpp`. In v3, we will hopefully deprecate the Puppeteer approach altogether (since it's quite a leaky abstraction), or have an option for BPP to download and set up the browser for you.
```